lib/ansible/config/base.yml

```
@@ -1573,21 +1573,11 @@ INTERPRETER_PYTHON:
   description:
   - Path to the Python interpreter to be used for module execution on remote targets, or an automatic discovery mode.
     Supported discovery modes are ``auto`` (the default), ``auto_silent``, ``auto_legacy``, and ``auto_legacy_silent``.
-    All discovery modes employ a lookup table to use the included system Python (on distributions known to include one),
-    falling back to a fixed ordered list of well-known Python interpreter locations if a platform-specific default is not
-    available. The fallback behavior will issue a warning that the interpreter should be set explicitly (since interpreters
+    All discovery modes match against an ordered list of well-known Python interpreter locations. 
+    The fallback behavior will issue a warning that the interpreter should be set explicitly (since interpreters
     installed later may change which one is used). This warning behavior can be disabled by setting ``auto_silent`` or
     ``auto_legacy_silent``. The value of ``auto_legacy`` provides all the same behavior, but for backward-compatibility
```

Don't suggest to use deprecated settings maybe?
```suggestion
    installed later may change which one is used). This warning behavior can be disabled by setting ``auto_silent``.
```